src/rust/src/wallet.rs

```diff
@@ -940,14 +941,20 @@ pub extern "C" fn orchard_wallet_append_bundle_commitments(
 #[no_mangle]
 pub extern "C" fn orchard_wallet_commitment_tree_root(
     wallet: *const Wallet,
+    checkpoint_depth: usize,
     root_ret: *mut [u8; 32],
-) {
+) -> bool {
     let wallet = unsafe { wallet.as_ref() }.expect("Wallet pointer may not be null");
     let root_ret = unsafe { root_ret.as_mut() }.expect("Cannot return to the null pointer.");
 
     // there is always a valid note commitment tree root at depth 0
     // (it may be the empty root)
```

Comment:
```suggestion
```
(comment is no longer applicable, even if it is still a true statement)

Reply:
I think that this comment is still relevant; `checkpoint_depth` will be 0, it's `anchorConfirmations` that must be >= 1.